tests/nnstreamer_decoder_pose/runTest.sh

```diff
@@ -30,4 +30,7 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc ! videoconvert ! video
 # THIS WON'T FAIL, BUT NOT MUCH MEANINGFUL.
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num_buffers=4 ! videoconvert ! videoscale ! video/x-raw,width=14,height=14,format=RGB ! tensor_converter ! tensor_split name=a tensorseg=1:14:14:1,2:14:14:1 a.src_0 ! tensor_transform mode=transpose option=1:2:0:3 ! tensor_decoder mode=pose_estimation option1=320:240 option2=14:14 ! fakesink" 1 0 0 $PERFORMANCE
 
+# TEST WITH MORE BUFFERS
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num_buffers=100 ! videoconvert ! videoscale ! video/x-raw,width=14,height=14,format=RGB ! tensor_converter ! tensor_split name=a tensorseg=1:14:14:1,2:14:14:1 a.src_0 ! tensor_transform mode=transpose option=1:2:0:3 ! tensor_decoder mode=pose_estimation option1=320:240 option2=14:14 option3=notused ! fakesink" 2 0 0 $PERFORMANCE
```

Comment:
Q. Which lines do you want to cover?

http://nnsuite.mooo.com/nnstreamer/ci/gcov_html/nnstreamer-1.5.2/ext/nnstreamer/tensor_decoder/tensordec-pose.c.gcov.html

Reply:
The function `draw_line_with_dot` is executed w.r.t. the value of `XYdata[i]->valid` (in line 382). But current testcase does not cover the execution of this function.
I checked the added one cover it at least in my environment.

Reply:
I see. Thanks.